app/src/main/java/com/ataulm/whatsnext/search/SearchPresenter.java

```diff
@@ -21,18 +19,14 @@
 
     private final WhatsNextService service;
     private final SearchDisplayer displayer;
-    private final TokensStore tokensStore;
```

Comment:
don't recall why this was here - probably to handle an error if the auth token expired.

This is a presenter and shouldn't handle that error anyway. Either it's handled silently before it gets here (use refresh token to get a new auth token), or in the case where the refresh token isn't valid the service should throw some descriptive error that the presenter can interpret as "go to Sign In".